### PR TITLE
migration of the prolog_trigger files to a separate repository

### PR DESCRIPTION
**Pull request by gianipez**
All the prolog_trigger files were migrated to https://github.com/gianipez/mu2e_trig_config/tree/main , which needs to be moved to the Mu2e project.

### CODE REVIEW

File: Trigger/src/PrescaleEvent_module.cc [hunk 1]
```diff
@@ -25,48 +28,68 @@ namespace mu2e
...
-    bool         useFilteredEvts_;
-    int          _debug;
-    //    TriggerFlag  _trigFlag;
-    unsigned     _nevt, _npass;
+    bool filter(art::Event & e) override;
+    virtual bool endRun(art::Run& run ) override;
+
+  private:
+
+    std::vector<EventModeConfig> _eventMode;
```
Comment (brownd1978) [inline Trigger/src/PrescaleEvent_module.cc:63]: Why is this a vector?  Can a single filter instance be used in multiple event modes?

**Review by brownd1978 — COMMENTED** [review]
As we discussed today, this fails the python build since it removes many of the includes the Offline trigger python depends on.

Also, since the new trigger prolog is independent of any existing prolog, this is the moment to make the cpr and tpr configurations consistent.

File: Trigger/src/PrescaleEvent_module.cc (hunk 1 above)
Comment (gianipez) [reply Trigger/src/PrescaleEvent_module.cc:63]: That's the current idea. The current design is to have the possibility to run the same trigger path in different event-modes with different prescale factors.

**Review by brownd1978 — COMMENTED** [review]
All the prolog from CRVResponse, which is a simulation package, please return that.
Similarly return all the conditions, like ConditionsService/data/czarnecki_Ti.tbl, these are not related to DAQ.  Let me know if we need a 2nd discussion to define the boundary conditions for what should/should not be migrated to common_config

**Comment by rlcee** [comment]
Dave's comment prompted to take a look and I see this is much more extensive than I thought it was.  
- how was this list made? It seems like you took ~every non-c++ file.  
- I don't understand why these files move (just a sampling of the question)
```
CRVConditions/data/nominal.txt
CaloConditions/data/nominal.txt
DbService/data/connections.txt
DbService/data/create.sql
GlobalConstantsService/data/ParticleList.txt
```
- do you have a plan for the organization of where these files go?
- I think I am not up to date on the plan, so not sure how to comment

**Comment by gianipez** [comment]
> All the prolog from CRVResponse was removed. That is a simulation package. Similarly conditions like ConditionsService/data/czarnecki_Ti.tbl are not related to DAQ, I don't understand why they were removed. Let me know if we need a 2nd discussion to define the boundary conditions for what should/should not be migrated to common_config

Apologies for the mistake. in ConditionsService/data/ there is a file I was typically including in the fcl (conditions_01.txt) and thus I moved the entire folder. For CRVRespnse, the issue was that I got mislead by the missing "MC" component in the folder name.

**Comment by brownd1978** [comment]
We need to talk about this PR.  I have a much clearer picture of what needs to be done, in particular:
- all fcl related to MonteCarlo must remain in Offline
- all fcl and data structures related to the Offline reconstruction exclusively must remain in Offline
- Offline build procedures and all the Musings need to be updated so that we can still run triggers in the Offline environment
- Affected modules  not currently using validated fcl need to be updated
Getting this right requires more work than simply moving directories but it will result in a more maintainable and logical structure.
If done right, I suspect common_config will no longer be necessary, we will only need trigger_config.
I suggest a meeting and go over each package affected by this PR one-by-one to understand what needs to be done.

**Comment by brownd1978** [comment]
Is there anything holding up processing this PR?

**Comment by gianipez** [comment]
> Is there anything holding up processing this PR?

Hi Dave,

I think I sent a slack message to set up a meeting to clarify the points you mentioned. Could you please take a look when you have time?

**Comment by rlcee** [comment]
If musings, proditions or validation are affected, please keep me in the loop

**Pull request closed without merging**